qiskit_experiments/library/calibration/half_angle_cal.py

```diff
@@ -38,10 +38,12 @@ def __init__(
         calibrations: Calibrations,
         backend: Optional[Backend] = None,
         schedule_name: str = "sx",
-        cal_parameter_name: Optional[str] = "amp",
+        cal_parameter_name: Optional[str] = "angle",
         auto_update: bool = True,
     ):
-        """see class :class:`HalfAngle` for details.
+        """see class :class:`HalfAngle` for details. The experiment updates the
+        angle of the complex amplitude represented in (amp,angle) representation,
+        matching the logic of the `ScalableSymbolicPulse` class.
```

Comment:
`ScalableSymbolicPulse` is too much details for experimentalist. You can just say _angle parameter of your pulse_ and `# section: note` if you want add more details.